charts/fybrik/values.yaml

```diff
@@ -79,7 +79,11 @@ coordinator:
 
   # Configure the vault instance to be used by the coordinator manager
   vault:
-    # Set to the Vault address. 
+    # WARNING: it's an advance feature, set it to "false", if you all your modules and connectors do not require getting
```

Comment:
advance->advanced
remove you + no comma before if ( set it to "false" only if all your modules...)

Reply:
you're right, fixed, thanks